test/integration/rebalancing_test.lua

```diff
@@ -168,7 +168,9 @@ function g.test()
 
     helpers.retrying({}, function() g.sA1:eval([[
         for _, f in pairs(require('fiber').info()) do
-            if f.name:startswith('vshard.') and f.name ~= 'vshard.conn_man' then
+            if f.name:startswith('vshard.') and not (f.name == 'vshard.conn_man'
```

Comment:
@Serpentian hello!
Does the fiber vshard.conn_man still exist in vshard? I haven't found it in the latest vshard code.